sdk/tests/deqp/functional/gles3/es3fNegativeTextureApiTests.js

```diff
@@ -125,6 +130,7 @@ goog.scope(function() {
 
         testGroup.addChild(new es3fApiCase.ApiCaseCallback('compressedteximage2d_invalid_target', 'Invalid gl.compressedTexImage2D() usage', gl,
         function() {
+            if (!haveCompressedTextureES30) { return; }
```

Comment:
Could you add a helper function like "etc2Unsupported()" which displays some debugging text? Like:

`function etc2Unsupported() {`
`  debug('Skipping test due to no WEBGL_compressed_texture_es3_0 support');`
`}`

then just call it before all of these return statements. It will help people understand what's going on in the future.